cmd/wallet-web/src/pages/OIDCShare.vue

```diff
@@ -237,7 +235,8 @@ export default {
       if (!e.message.includes('12009')) {
         this.errors.push('Error initiating credential share');
       }
-      console.error('initiating credential share failed,', e);
```

Comment:
I think we should keep this for debugging purposes since we don't log this error anywhere else

Reply:
updated

Reply:
Can we keep the console.error on the same line as before?